features/promotion/managing_catalog_promotions/prioritizing_catalog_promotions.feature

```diff
@@ -0,0 +1,74 @@
+@managing_catalog_promotions
+Feature: Prioritizing a catalog promotion
+    In order to avoid the same catalog promotion priority
+    As an Administrator
+    I want to set distinct priority on multiple catalog promotions
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is a catalog promotion "Winter sale" with priority 20
+        And there is a catalog promotion "Year-end sale" with priority 30
+        And there is a catalog promotion "Spring sale" with priority 40
+        And I am logged in as an administrator
+
+    @api @ui
+    Scenario: Adding a catalog promotion with a priority higher than all existing catalog promotions does not change priority values
+        When I create a new catalog promotion with "collection_sale" code and "Collection sale" name and 50 priority
+        Then there should be 4 catalog promotions on the list
+        And the catalog promotion named "Winter sale" should have priority 20
+        And the catalog promotion named "Year-end sale" should have priority 30
+        And the catalog promotion named "Spring sale" should have priority 40
+        And the catalog promotion named "Collection sale" should have priority 50
+
+    @api @ui
+    Scenario: Adding a catalog promotion with a priority lower than all existing ones
+              increases the priority value of other catalog promotions by 1
```

Comment:
I am not sure if we need these details, in the scenario we are describing what we do, not an effect, IMO assertions are enough to understand priorities logic :)